src/js/control-bar/volume-menu-button.js

```diff
@@ -58,13 +58,41 @@ class VolumeMenuButton extends MenuButton {
     updateVisibility.call(this);
     this.on(player, 'loadstart', updateVisibility);
 
-    this.on(this.volumeBar, ['slideractive', 'focus'], function(){
-      this.addClass('vjs-slider-active');
+    this.toggleMutedLabel();
+
+    this.on('focus', function() {
+      this.el_.setAttribute('aria-expanded', 'true');
     });
 
-    this.on(this.volumeBar, ['sliderinactive', 'blur'], function(){
-      this.removeClass('vjs-slider-active');
+    let playerEl = player.el();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).